### ISSUE TITLE
Docker image build error while fetching alpinelinux packages

### ISSUE THREAD

**Issue by AbyssCoreInc** [dekiesel/arm7v32_timescaledb#2]
I ran into this problem when trying to build docker image: 
```

 ---> Running in b2afb3ddc518
+ apk update
fetch https://dl-cdn.alpinelinux.org/alpine/v3.13/main/armv7/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.13/community/armv7/APKINDEX.tar.gz
ERROR: https://dl-cdn.alpinelinux.org/alpine/v3.13/main: temporary error (try again later)
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.13/main: No such file or directory
ERROR: https://dl-cdn.alpinelinux.org/alpine/v3.13/community: temporary error (try again later)
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.13/community: No such file or directory
2 errors; 15 distinct packages available

```
with wget those addresses work just fine. I am trying to build this with Raspberry pi 3 running raspbian 64bit version.
Also docker pull from docker hub does not seem to be working and I get this error:

```
sudo docker pull dekiesel/timescaledb-postgis
Using default tag: latest
Error response from daemon: manifest for dekiesel/timescaledb-postgis:latest not found: manifest unknown: manifest unknown
```
Any ideas what could cause this problem and how to get around it?

**Comment by dekiesel** [comment]
The first error is definitely weird, but the second one is simple: You are trying to find a 64bit manifest, but this image only has a 32 bit manifest :) Doesn't the official timescaledb-image work for you?

**Comment by AbyssCoreInc** [comment]
Yeah that first one is weird. I got that with 32bit raspbian as well. I did not know that docker pull was architecture specific as I got intel binaries earlier as well, which obviously did not work... For that reason the stock image does not work as it pulls intel binaries which give this error:
```
standard_init_linux.go:228: exec user process caused: exec format error
```
That is why I think it is necessary to compile ARM version. 32bit should work at least better than i386 version...
